src/constants.jl

```diff
@@ -19,6 +20,7 @@ const COMMANDS_NAMES = LittleDict([
     :tz => :time_zone,
     :discourse => :discourse,
     :src => :source,
+    :con => :julia_con,
```

Comment:
I guess `juliacon` looks better. It's up to you :-)

Reply:
Now that you say it ... ^^